src/views/common/DashOverview.tsx

```diff
@@ -51,7 +51,12 @@ export const DashOverviewIntro: React.FC<{
     MIN_SAFE_HEALTH_FACTOR
   );
 
-  const limitAmount = balance && maxAmount?.lt(balance) ? maxAmount : balance;
+  const limitAmount =
+    balance && maxAmount?.lt(balance)
+      ? maxAmount.gt(0)
```

Comment:
You forgot one spot ^^
```suggestion
      ? maxAmount.gt(constants.Zero)
```